client/selfrepair/normandy_driver.js

```diff
@@ -117,6 +117,9 @@ export default class NormandyDriver {
         },
         sync(data) {
           client.syncSetup = data.setup;
+          client.syncDesktopDevices = data.desktopDevices;
+          client.syncMobileDevices = data.mobileDevices;
+          client.syncTotalDevices = data.totalDevices;
```

Comment:
These should have `|| 0` at the end, in case this runs on a Firefox that doesn't have the UITour patches.